code/controllers/subsystem/aura.dm

```diff
@@ -91,8 +91,10 @@ SUBSYSTEM_DEF(aura)
 	var/hive_number = XENO_HIVE_NORMAL
 	///Whether we should skip the next tick. Set to false after skipping once. Won't pulse to targets or reduce duration.
 	var/suppressed = FALSE
+	///If theres anything in this list, only give our aura to specific caste types in this list
+	var/list/specific_castes = list()
```

Comment:
ok looking again this var doesnt need to exist since you can just handle this on can_receive_aura, no?

Reply:
puppeteer is by design not supposed to be allowed to give auras to other xenos so if i did they would be able to

Reply:
ok this isnt relevant anymore